shogun-boot/src/main/resources/templates/index.html

```diff
@@ -167,6 +180,9 @@ <h3 class="pb-2 pt-4">Applications</h3>
           ? app.clientConfig.description
           : ''
       }));
+    if (!keycloak.authenticated || !applications) {
+      document.querySelector('#app-label-visibility').hidden = false;
+    }
```

Comment:
Indentation seems off and I'm, not quite sure if I get this right, but actually this would _show_ the info message if the user is not logged in or no applications are present? But is there any code/style that hides the message?